src/commandHandlers/help.ts

```diff
@@ -19,15 +19,8 @@ export const command = async (arg: [string, string], embed: MessageEmbed) => {
       false
     );
   } else {
-    let text = '';
-    commandList.forEach((commandItem) => {
-      text = text
-        .concat(`**${COMMAND_PREFIX}${commandItem.triggers[0]}**  `)
-        .concat(`${commandItem.description}\n`)
-        .concat(
-          'Usage:' + '`' + `${COMMAND_PREFIX}${commandItem.usage}\n` + '`'
-        );
-    });
+    const format = commandItem => `**${COMMAND_PREFIX}${commandItem.triggers[0]}**`;
+    const text = commandList.map(format).join(" | ");
     embed.addField('\u200b', text);
   }
```

Comment:
```suggestion
    const format = commandItem =>
     `**${COMMAND_PREFIX}${commandItem.triggers[0]}**`;
    const text = commandList.map(format).join(' | ');
    embed.addField('\u200b', text);
  }
```
Should fix the lint errors.